src/4D_api.cpp

```diff
@@ -653,6 +656,38 @@ indicator, as in {"+proj=utm", "+zone=32"}, or leave it out, as in {"proj=utm",
     return P;
 }
 
+/*************************************************************************************/
+PJ *proj_create_argv_internal (PJ_CONTEXT *ctx, int argc, char **argv) {
```

Comment:
Is the only difference from this one and the public counter-part that this one calls `proj_create_internal`?

Reply:
yes

Reply:
Adressed by renaming proj_create_internal() to pj_create_internal() and proj_create_argv_internal() to pj_create_argv_internal()